tools/cli/commands/rsync.js

```diff
@@ -228,6 +228,9 @@ async function addVendorFilesToFilter( source, filters ) {
  */
 async function rsyncToDest( source, dest, pluginDestPath ) {
 	const filters = new Set();
+
+	// Exclude git internals
```

Comment:
Let's make this clearer about what we're doing.
```suggestion
	// Exclude any `.git` dirs, mostly in case someone ran composer with --prefer-source (or composer fell back to that).
```